scripts/get-translations.ts

```diff
@@ -0,0 +1,106 @@
+import Airtable from 'airtable'
+import { set as lodashSet } from 'lodash'
+import { writeFileSync, existsSync, mkdirSync } from 'fs'
+import dotenv from 'dotenv'
+import { default as AirtableRecord } from 'airtable/lib/record'
+
+dotenv.config()
+
+enum AirTableColumn {
+  Key = 'id',
+  Czech = 'cs',
+  English = 'en',
+}
+interface AirtableStructure {
+  [AirTableColumn.Key]: string
+  [AirTableColumn.Czech]: string
+  [AirTableColumn.English]: string
+}
+
+const config = {
+  KEY: process.env.AIRTABLE_TRANSLATION_KEY,
+  BASE_ID: process.env.AIRTABLE_TRANSLATION_BASE,
+  TABLE_NAME: process.env.AIRTABLE_TRANSLATION_BASE_NAME || 'Translations',
+  VIEW: process.env.AIRTABLE_TRANSLATION_VIEW || 'Grid view',
+}
+
+interface MappedTranslations {
+  lang: string
+  key: string
+  translation: string
+}
+
+const getAirtableRecords = async (): Promise<AirtableRecord[]> => {
+  if (!config.KEY || !config.BASE_ID) {
+    throw new Error('Env variables not specified!')
+  }
+
+  return new Airtable({ apiKey: config.KEY })
```

Comment:
Díky za refactoring. Mohl bych ještě poprosit o úpravu na `new Airtable({ apiKey: config.KEY })
    .base(config.BASE_ID).table(config.TABLE_NAME)`? Je to ještě více přehledné řešení.

Reply:
rozdelim pro prehlednost, `.table()` zavolat nejde, ale poupravim